technic_chests/inventory.lua

```diff
@@ -67,6 +69,37 @@ function technic.chests.sort_inv(inv, mode)
 			end
 			return a.key < b.key
 		end)
+	elseif mode == 4 then
+		-- Natural
+		do -- Collect item stacks for sorting
```

Comment:
What does `do`, umm... do?

Reply:
It is just scope closure for scope control, same as { } in javascript or c/c++
So not required but added to separate collector and sort algorithm for readability

Reply:
Interesting...